Source/Python/WarpX_py.H

```diff
@@ -10,23 +10,6 @@
 
 #include "WarpXWrappers.H"
 
-
-extern "C" {
-
-    extern WARPX_CALLBACK_PY_FUNC_0 warpx_py_afterinit;
-    extern WARPX_CALLBACK_PY_FUNC_0 warpx_py_beforeEsolve;
-    extern WARPX_CALLBACK_PY_FUNC_0 warpx_py_poissonsolver;
-    extern WARPX_CALLBACK_PY_FUNC_0 warpx_py_afterEsolve;
-    extern WARPX_CALLBACK_PY_FUNC_0 warpx_py_beforedeposition;
-    extern WARPX_CALLBACK_PY_FUNC_0 warpx_py_afterdeposition;
-    extern WARPX_CALLBACK_PY_FUNC_0 warpx_py_particlescraper;
-    extern WARPX_CALLBACK_PY_FUNC_0 warpx_py_particleloader;
-    extern WARPX_CALLBACK_PY_FUNC_0 warpx_py_beforestep;
-    extern WARPX_CALLBACK_PY_FUNC_0 warpx_py_afterstep;
-    extern WARPX_CALLBACK_PY_FUNC_0 warpx_py_afterrestart;
-    extern WARPX_CALLBACK_PY_FUNC_0 warpx_py_particleinjection;
-    extern WARPX_CALLBACK_PY_FUNC_0 warpx_py_appliedfields;
-
-}
+extern std::map< std::string, WARPX_CALLBACK_PY_FUNC_0 > warpx_callback_py_map;
```

Comment:
```suggestion
/** let's add a doc string here since this one encodes global state of the library.
 *
 * Can be multi-line, etc. as usual :)
 */
extern std::map< std::string, WARPX_CALLBACK_PY_FUNC_0 > warpx_callback_py_map;
```